submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/common/BaseIntegrationTest.kt

```diff
@@ -12,15 +12,18 @@ internal open class BaseIntegrationTest(private val tempFolder: TemporaryFolder)
 
     @BeforeAll
     fun init() {
-        val dropbox = tempFolder.createDirectory("dropbox")
         val temp = tempFolder.createDirectory("tmp")
         submissionPath = "${tempFolder.root.absolutePath}/submission"
         ftpPath = "${tempFolder.root.absolutePath}/ftpPath"
 
         System.setProperty("app.submissionPath", submissionPath)
         System.setProperty("app.ftpPath", ftpPath)
         System.setProperty("app.tempDirPath", temp.absolutePath)
+
+        val dropbox = tempFolder.createDirectory("dropbox")
+        val magicFolders = tempFolder.createDirectory("magic")
```

Comment:
I'm disliking more and more this `magicFolder` name... Perhaps we could think about using something like `userSecretFolder` or something more meaningful but that could be another ticket.